samples/esb/esb_ptx/README.rst

```diff
@@ -1,13 +1,34 @@
-.. _esb_prx_ptx:
+.. _esb_ptx:
 
-Enhanced ShockBurst: Transmitter/Receiver
-#########################################
+Enhanced ShockBurst: Transmitter
+################################
 
 .. contents::
    :local:
    :depth: 2
 
-The Enhanced ShockBurst Transmitter/Receiver sample shows the basic steps that are needed to transmit and receive packets using :ref:`ug_esb`.
+This sample shows the Enhanced ShockBurst Transmitter application.
+It includes the basic steps that are needed to transmit and receive packets using :ref:`ug_esb`.
+Follow the instructions and use the files from this page, and the :ref:`esb_prx` sample.
+
+Requirements
+************
+
+.. note::
+   Follow the steps from and include the :ref:`esb_prx` sample with this sample.
+
+.. esb_ptx_sample_requirements_start
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+You can use any two of the development kits listed above and mix different development kits.
+
+.. esb_ptx_sample_requirements_end
+
+
+.. esb_ptx_sample_remaining_start
 
 Overview
 ********
```

Comment:
In short: the sample in the NCS is meant to showcase 1 library/feature and run as a standalone application (meant as a "user application", not an application in the NCS). One should be able to configure and program it alone, and if it needs another sample for testing, that should be stated in the requirements and mentioned in the Testing section. We cannot bundle together samples, because that makes them an application (in the NCS terms).